### DISCUSSION TITLE
Parallel Compilation

### DISCUSSION THREAD

**Discussion by gianks** [zephir-lang/zephir#2271] [Q&A] [answered]
Hi, 
is there a way to speed-up compilation time of phalcon with parallel processing as in -j for gcc/g++?
If not, this is surely a desirable feature for Zephir.

Thanks

**Comment by Jeckerson** [comment]
Hello, yes

Use `-j` option: `php zephir compile -j 32`

It is available since `0.13.1` - https://github.com/zephir-lang/zephir/blob/development/CHANGELOG.md#0131---2021-03-31

**Comment by gianks** [comment, marked as answer]
Thanks for the hint, i could have tried on my own i guess.
This option should be implemented within build too IMHO.

I suggest to modify a little the help of the command to suggest you can pass extra arguments.
I don't think people go often thru changelogs to discover features :)

Thanks for the nice work!

**Reply by Jeckerson** [reply]
Hello, could you please submit the issue with NFR?
